### ISSUE TITLE
Update to 1.6.1 Couchdb

### ISSUE THREAD

**Issue by Apcio** [felixge/couchdb-benchmarks#1]
Hello.
Is possible to update benchmark to 1.6.1 CouchDB ?
Server returns 415 code - probably there is no Header: Content-Type: application/json.

I do not have php skills for update this code, then I would like to ask for this.

//Edit:
Well it wasn't so hard to add headers to curl parameter.
I have learned a little about git and push a commit:
https://github.com/felixge/couchdb-benchmarks/pull/2

**Comment by felixge** [comment]
Yes, it's possible - but I haven't touched this code since 2009 - so don't hold your breath :)
